docs/user/admin/settings.rst

```diff
@@ -109,6 +109,15 @@ Result set::
 	  },
 	  "status" : 400
 	}
+	Query failed on both V1 and V2 SQL parser engines. V2 SQL parser error following:
+	{
+	  "error": {
+	    "reason": "Invalid SQL query",
+	    "details": "Failed to parse query due to offending symbol [DELETE] at: 'DELETE' <--- HERE... More details: Expecting tokens in {<EOF>, 'DESCRIBE', 'SELECT', 'SHOW', ';'}",
+	    "type": "SyntaxCheckException"
+	  },
+	  "status": 400
+	}
```

Comment:
1.Comparing with previous error message, the new one is misleading. 
2.Why the V2 engine throw syntax exception?  SQLFeatureDisabledException means user already disable sql plugin, then query will not executed in SQL engine.

Reply:
Good catch on this bug. Looks like the error string value for the V2 engine was being used. I will update the test and clear the error message with each attempt at the V1 and V2 engines.